kratos/python_scripts/auxiliar_methods_adaptative_remeshing.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+
+def AdaptativeRemeshingRunSolutionLoop(analysis):
+    """This function executes the solution loop of the AnalysisStage
+    It can be overridden by derived classes
+    """
+
+    # If we remesh using a process
+    computing_model_part = analysis._GetSolver().GetComputingModelPart()
+    root_model_part = computing_model_part.GetRootModelPart()
+
+    while analysis.time < analysis.end_time:
+        analysis.time = analysis._GetSolver().AdvanceInTime(analysis.time)
+        # We reinitialize if remeshed previously
+        if root_model_part.Is(KratosMultiphysics.MODIFIED):
```

Comment:
If we add this mayor functionalitiy here then I think it would make sense to add a `REMESHED` flag to indicate exactly what is going on

Reply:
We cannot adds many more flags...

Reply:
Thats why it should be discussed
For a major functionality I think it should ate least be considered

Reply:
Ok, but take into account that many implementations (not just python, but cpp) assume that flag, and should be changed